src/NServiceBus.Core/obsoletes-v8.cs

```diff
@@ -1445,4 +1445,33 @@ public static void DisablePublishing<T>(this TransportExtensions<T> transportExt
     }
 }
 
+namespace NServiceBus.Pipeline
+{
+    public partial interface ITransportReceiveContext
+    {
+        /// <summary>
+        /// Allows the pipeline to flag that it has been aborted and the receive operation should be rolled back.
+        /// </summary>
+        [ObsoleteEx(
+            Message = "The AbortReceiveOperation method has been removed. See the upgrade guide for more details.",
```

Comment:
This results in the attribute saying:

> The AbortReceiveOperation method has been removed. See the upgrade guide for more details. Will be removed in version 9.0.0.

I don't think we can say "has been removed" and "will be removed" in the same sentence.

Do we need more than this?

```suggestion
            Message = "See the upgrade guide for more details.",
```